ArduCopter/mode_guided.cpp

```diff
@@ -39,16 +40,19 @@ struct Guided_Limit {
     Vector3f start_pos; // start position as a distance from home in cm.  used for checking horiz_max limit
 } guided_limit;
 
-// guided_init - initialise guided controller
+// init - initialise guided controller
 bool ModeGuided::init(bool ignore_checks)
 {
     // start in position control mode
-    pos_control_start();
+    velaccel_control_start();
```

Comment:
I wonder why we are changing the default submode that Guided starts in from position control to VelAccel.  If we really want to do this then we should change the comment above this line.